src/lib/sync-profile.js

```diff
@@ -77,6 +77,10 @@ class SyncProfile {
     this.peer = peer
   }
 
+  getSw () {
```

Comment:
no need to set a getSw method, you can already directly access this.sw

Reply:
yes, that's right.